src/Phinx/Db/Adapter/MysqlAdapter.php

```diff
@@ -1000,6 +1001,8 @@ public function getSqlType($type, $limit = null)
                 return ['name' => 'bit', 'limit' => $limit ?: 64];
             case static::PHINX_TYPE_SMALL_INTEGER:
                 return ['name' => 'smallint', 'limit' => $limit ?: 6];
+            case static::PHINX_TYPE_MEDIUM_INTEGER:
```

Comment:
Shouldnt this be above small? 8, 6, 4

Reply:
We can change, but since we are returning on each case check, should not make a difference, should it?

Reply:
hahaha Know the feeling mate. Now, dumb question from someone who is doing their first github pull request, do I need to do another commit on my fork and another pull request or can I edit here to fix it?

Reply:
A commit on top is fine

Reply:
Done. I also moved the big int that was after the integer statement.